addons/l10n_sg_ubl_pint/models/account_move.py

```diff
@@ -0,0 +1,14 @@
+import uuid
+
+from odoo import models
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    def _l10n_sg_get_uuid(self):
+        """ The rule is only to provide a uuid. To keep it as simple as possible, we won't store it but instead generate it """
```

Comment:
the docstring is not super clear. Maybe something like: "SG Pint requires us to generate a uuid, to avoid storing a new field on the move, we derive it from the dbuuid and the move id." is clearer ?